script/Arcueid/Day 5/05_00_ARC05_1C.txt

```diff
@@ -257,22 +257,22 @@ I've got some questions I need to ask her as well, so I'll just talk with her fo
 [sha:8c2aa714e968a5e2f43ebb2eb4dff1e561256fb0]{
 -- Page 162, Offset 8981.
 -- 「……そうね。やっぱりそういう会話になるわよね、わたしと志貴の場合」
-"...That's right. Pretty natural you'd want to talk about that after what you saw."
+"Sure, I guess. I guess that's exactly the kind of thing you'd wanna talk about, after what happened to us."
 }
 [sha:d2e807deacbd0a411ca6160ed3190f2dcc7bf877]{
 -- Page 163, Offset 8982.
 -- 「昨日のあいつ……ヴローヴとか言っていたけど、あいつは一体なんなんだ。
-"That guy from yesterday... I believe he called himself Vlov――what the hell is he?
+"That guy yesterday... I believe he called himself Vlov――what the hell is he?
 }
 [sha:09d9b21606f813377f06695bc9af6aa2f9cfc316]{
 -- Page 163, Offset 8983.
 -- 吸血鬼なのは分かる。仰々しい服とか、おまえよりよっぽどらしかったからな。けど―――」
-I understand that he's a vampire. That grandiose outfit makes him look much more vampire-like than you after all. But―――"
+I get that he's a vampire, I mean he's got those elaborate clothes on and all, so he looks way more vampire-like than you at least.
 }
 [sha:ad8345166d09815cbec61df3f15d27036ccd5980]{
 -- Page 164, Offset 8984.
 -- あの青色の炎。廊下に充満し、人間だけを燃やすあの火はなんなのか。あれは吸血鬼のイメージとはかけ離れすぎている。
-That blue flame. What was that flame that filled the hallway and burned humans but left everything else intact? That didn't fit with how I picture vampires at all.
+Those blue flames... spreading through the hallway, burning only humans in their wake... What are they? That's not how I imagine vampires are like at all.
```

Comment:
That's now how I imagine vampires being like at all.

Reply:
> burning only humans in their wake

Generally the wake is used to refer to things _left behind_, not sure it sits right with an active construction like this